card/mod/05_standard/set/type/file.rb

```diff
@@ -87,9 +87,11 @@ def preview  args
   end
 
   view :preview_editor, tags: :unknown_ok do |args|
+    cached_upload_card_name = Card::Env.params[:attachment_card_name]
+    cached_upload_card_name.gsub!(/\[\w+\]$/, "[cached_upload]")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.